src/PyCall.jl

```diff
@@ -28,14 +28,6 @@ if VERSION >= v"0.4.0-dev+3710"
 else
     const unsafe_convert = Base.convert
 end
-# This is not the exact version but should be closed enough
-if VERSION >= v"0.4.0-dev+4922"
```

Comment:
............... In this case should I add back my other commit (which I'm keeping on a backup branch right now) so that those people don't get a segfault on exit?